lib/blacklight/search_builder.rb

```diff
@@ -1,140 +1,6 @@
 module Blacklight
   class SearchBuilder
-    extend Deprecation
```

Comment:
I still don't understand why this can't be a class, and have `::SearchBuilder` extend it if it needs to.